json/Name_Chip_ActiveName.txt

```diff
@@ -2722,322 +2722,322 @@
 		"jp_text": "疾弾斬１",
-		"tr_text": ""
+		"tr_text": "Quickshot Cutter 1"
 	},
 	{
 		"assign": "60046",
 		"jp_text": "疾弾斬１＋",
-		"tr_text": ""
+		"tr_text": "Quickshot Cutter 1+"
 	},
 	{
 		"assign": "60047",
 		"jp_text": "疾弾斬２",
-		"tr_text": ""
+		"tr_text": "Quickshot Cutter 2"
 	},
 	{
 		"assign": "60048",
 		"jp_text": "疾弾斬２＋",
-		"tr_text": ""
+		"tr_text": "Quickshot Cutter 2+"
 	},
 	{
 		"assign": "60049",
 		"jp_text": "疾弾斬３",
-		"tr_text": ""
+		"tr_text": "Quickshot Cutter 3"
 	},
 	{
 		"assign": "60050",
 		"jp_text": "疾弾斬３＋",
-		"tr_text": ""
+		"tr_text": "Quickshot Cutter 3+"
 	},
 	{
 		"assign": "60051",
 		"jp_text": "疾弾斬ＥＸ",
-		"tr_text": ""
+		"tr_text": "Quickshot Cutter EX"
 	},
 	{
 		"assign": "60052",
 		"jp_text": "疾弾斬ＥＸ＋",
-		"tr_text": ""
+		"tr_text": "Quickshot Cutter EX+"
 	},
 	{
 		"assign": "60053",
 		"jp_text": "飛氷ＥＸ",
-		"tr_text": ""
+		"tr_text": "Flying Ice EX"
 	},
 	{
 		"assign": "60054",
 		"jp_text": "飛氷ＥＸ＋",
-		"tr_text": ""
+		"tr_text": "Flying Ice EX+"
 	},
 	{
 		"assign": "60055",
 		"jp_text": "猛襲刃１",
-		"tr_text": ""
+		"tr_text": "Furious Blades 1"
 	},
 	{
 		"assign": "60056",
 		"jp_text": "猛襲刃１＋",
-		"tr_text": ""
+		"tr_text": "Furious Blades 1+"
 	},
 	{
 		"assign": "60057",
 		"jp_text": "猛襲刃２",
-		"tr_text": ""
+		"tr_text": "Furious Blades 2"
 	},
 	{
 		"assign": "60058",
 		"jp_text": "猛襲刃２＋",
-		"tr_text": ""
+		"tr_text": "Furious Blades 2+"
 	},
 	{
 		"assign": "60059",
 		"jp_text": "猛襲刃３",
-		"tr_text": ""
+		"tr_text": "Furious Blades 3"
 	},
 	{
 		"assign": "60060",
 		"jp_text": "猛襲刃３＋",
-		"tr_text": ""
+		"tr_text": "Furious Blades 3+"
 	},
 	{
 		"assign": "60061",
 		"jp_text": "猛襲刃ＥＸ",
-		"tr_text": ""
+		"tr_text": "Furious Blades EX"
 	},
 	{
 		"assign": "60062",
 		"jp_text": "猛襲刃ＥＸ＋",
-		"tr_text": ""
+		"tr_text": "Furious Blades EX+"
 	},
 	{
 		"assign": "60063",
 		"jp_text": "極一蹴１",
-		"tr_text": ""
+		"tr_text": "Ultimate Kick 1"
 	},
 	{
 		"assign": "60064",
 		"jp_text": "極一蹴１＋",
-		"tr_text": ""
+		"tr_text": "Ultimate Kick 1+"
 	},
 	{
 		"assign": "60065",
 		"jp_text": "極一蹴２",
-		"tr_text": ""
+		"tr_text": "Ultimate Kick 2"
 	},
 	{
 		"assign": "60066",
 		"jp_text": "極一蹴２＋",
-		"tr_text": ""
+		"tr_text": "Ultimate Kick 2+"
 	},
 	{
 		"assign": "60067",
 		"jp_text": "極一蹴３",
-		"tr_text": ""
+		"tr_text": "Ultimate Kick 3"
 	},
 	{
 		"assign": "60068",
 		"jp_text": "極一蹴３＋",
-		"tr_text": ""
+		"tr_text": "Ultimate Kick 3+"
 	},
 	{
 		"assign": "60069",
 		"jp_text": "極一蹴ＥＸ",
-		"tr_text": ""
+		"tr_text": "Ultimate Kick EX"
 	},
 	{
 		"assign": "60070",
 		"jp_text": "極一蹴ＥＸ＋",
-		"tr_text": ""
+		"tr_text": "Ultimate Kick EX+"
 	},
 	{
 		"assign": "60071",
 		"jp_text": "炎渦ＥＸ",
-		"tr_text": ""
+		"tr_text": "Flame Vortex EX"
 	},
 	{
 		"assign": "60072",
 		"jp_text": "炎渦ＥＸ＋",
-		"tr_text": ""
+		"tr_text": "Flame Vortex EX+"
 	},
 	{
 		"assign": "60073",
 		"jp_text": "極大断１",
-		"tr_text": ""
+		"tr_text": "Ultimate Cutter 1"
```

Comment:
Not entirely sure about this one as I think I've used both "Ultimate" and "Cutter" elsewhere for different kanji.